tools/c7n_mailer/c7n_mailer/slack_delivery.py

```diff
@@ -15,21 +15,20 @@
 
 from botocore.vendored import requests
```

Comment:
I think this needs to switch to normal requests rather than botocore, we can't depend on AWS SDK